SPECS/tensorflow/tensorflow.spec

```diff
@@ -64,6 +65,43 @@ Requires:       python3-zipp
 %description -n python3-tensorflow
 Python 3 version.
 
+%package -n python3-tf-nightly
```

Comment:
What does this package do?

Reply:
to remove the circular dep among the packages had to include this.